compiler/runtime/Makefile

```diff
@@ -1,8 +1,12 @@
 MEMSIZE32=1073741824
 
+COMMONFLAGS=-Wall -Wextra -Wno-sign-conversion -Wno-sometimes-uninitialized -Wno-c2x-extensions -Wsign-compare -Wextra-semi-stmt
```

Comment:
I would probably throw in -Wconversion too. That tends to catch a lot of useful things.

Reply:
:+1: 

I actually did `-Weverything` and looked at warnings that appeared and could be worth fixing later.